dev/.rat-excludes

```diff
@@ -106,3 +106,4 @@ spark-warehouse
 structured-streaming/*
 kafka-source-initial-offset-version-2.1.0.bin
 kafka-source-initial-offset-future-version.bin
+vote.tmpl
```

Comment:
even if rat doesn't check, isn't vote.tmpl packaged into the source release this way?

Reply:
for example, `.git` is removed from release here https://github.com/apache/spark/blob/master/dev/create-release/release-build.sh#L157

Reply:
Are you saying this file should not be packaged in the source release? Not sure I see why that would be the case. There's a lot of stuff in `.rat-excludes` that is still packaged.